### PR TITLE
initial commit FE + login form

### PR DESCRIPTION
**Pull request by alexdarahan**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Warhip Battle</title>
```
Comment (evglitvin) [inline frontend/public/index.html]: Warship

Comment (alexdarahan) [reply frontend/public/index.html]: done

File: frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (smushtyn) [inline frontend/public/index.html]: Is meta going to be changed?

Comment (alexdarahan) [reply frontend/public/index.html]: yes)

**Pull request merged**